components/horn-client/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+#*******************************************************************************
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#******************************************************************************/
+
+[package]
+name = "horn-client"
+version = "0.1.0"
+edition = "2021"
```

Comment:
is there a particular reason why some crates use edition 2018 and some others use 2021?

Reply:
No, there is no reason. I will adapt it to version 2021.